python_modules/libraries/dagster-dbt/dagster_dbt/dbt_artifacts.py

```diff
@@ -0,0 +1,164 @@
+                self._target_folder, "manifest.json"
+            )
+        else:
+            self._manifest_path = self._prepare_manifest()
+
+    @public
+    @property
+    def manifest_path(self) -> Path:
+        """The path to the manifest.json, compiling the manifest first if in development or
+        ensuring it already exists if not.
+        """
+        if not self._manifest_path.exists():
+            if self._should_use_built_artifacts():
+                raise DagsterDbtManifestNotPreparedError(
+                    f"Did not find prepared manifest.json at expected path {self._manifest_path}.\n"
+                    "If this is an environment that is expected to prepare the manifest at run time, "
+                    "set the environment variable DAGSTER_DBT_PARSE_PROJECT_ON_LOAD."
+                )
+            else:
+                raise DagsterDbtManifestNotPreparedError(
+                    f"Unexpected state, {self._manifest_path} should have been created when this "
+                    "object was initialized."
+                )
+
+        return self._manifest_path
+
+    @public
+    def prepare(self, *, quiet=False) -> None:
+        """A method that can be called as part of the deployment process to handle
+        preparing the manifest and if packaged_project_dir is set, handle copying
+        the dbt project to that directory.
+
+        Args:
+            quiet (bool):
+                Disable logging
+                Default: False
+        """
+        level = logging.DEBUG if quiet else logging.INFO
+        self._prepare_manifest(level)
+        self._handle_package_data(level)
+        logger.log(level, "Preparation complete.")
+
+    @public
+    def get_cli_resource(self, **kwargs) -> DbtCliResource:
+        """Construct a DbtCliResource that targets the correct project directory, passing any
+        kwargs to DbtCliResource.
+        If packaged_project_dir is set, resolves to that outside of the development context and
+        uses project_dir when in a development context.
+        """
+        return DbtCliResource(
+            project_dir=os.fspath(self._current_project_dir),
+            **kwargs,
+        )
+
+    def _should_use_built_artifacts(self) -> bool:
+        # use built artifacts unless
+        return not (
+            # launched via `dagster dev` cli
+            bool(os.getenv("DAGSTER_IS_DEV_CLI"))
+            or
+            # or if explicitly opted in to loading at runtime
+            bool(os.getenv("DAGSTER_DBT_PARSE_PROJECT_ON_LOAD"))
+        )
+
+    @property
+    def _current_project_dir(self) -> Path:
+        if self._should_use_built_artifacts():
+            return self._packaged_project_dir if self._packaged_project_dir else self.project_dir
+
+        return self.project_dir
+
+    def _prepare_manifest(self, level: int = logging.DEBUG) -> Path:
+        logger.log(level, f"Preparing dbt artifacts in {self.project_dir}.")
```

Comment:
Mention the path where artifacts are being created.

```suggestion
        logger.log(level, f"Preparing dbt artifacts for {self.project_dir} at {self._target_folder}.")
```